app/views/components/_contents-list.html.erb

```diff
@@ -16,20 +16,47 @@
       <%= t("content_item.contents") %>
     </h2>
     <ol class="app-c-contents-list__list">
-      <% contents.each do |contents_item| %>
+      <% contents.each.with_index(1) do |contents_item, position| %>
+        <%
+          active_class = contents_item[:active]? "app-c-contents-list__list-item--active" : ""
+          aria_current = contents_item[:active]? "aria-current = true" : ""
```

Comment:
We don't really need a value for aria_current is active is false, so could we rewrite this as:

` aria_current = "aria-current = true" if contents_item[:active]`

Same goes for active class

Reply:
Done.